src/main/java/com/google/sps/data/VideoState.java

```diff
@@ -0,0 +1,24 @@
+package com.google.sps.data;
+
+public enum VideoState {
```

Comment:
move inside Video class because this is always in reference to the Video object.